docs/server.rst

```diff
@@ -115,6 +115,13 @@ Also the following Debian packages are required:
 * ``python-dev``
 * ``libevent-dev``
 
+Debian 10 required following packages:
+* ``iproute2``
+* ``bridge-utils``
+* ``python-dev``
+* ``libevent-dev``
+* ``libnetfilter-conntrack-dev``
```

Comment:
This packet is not required any more as far as I know -- we do not have it installed on our servers and tunneldigger works fine there.